about.html

```diff
@@ -0,0 +1,148 @@
+                </div>
+                <div class="logoo">
+                    <div class="logoimg"><a href="./home.html"><img src="./images/thelogo.png" alt="logo" width="100px" height="90px"></a></div>
+                    <div class="text">
+                        <p>Click logo!</p>
+                    </div>
+                </div>
+            </div>
+
+            <!-- mobile navigation  -->
+            <div class="list_nav">
+                <ul>
+                    <li><a href="./home.html">Home</a></li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#project">Project</a></li>
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#sponsor">Sponsor</a></li>
+                    <li><a href="#news">News</a></li>
+                </ul>
+                <button type="submit" class="campaign">Learn more</button>
+            </div>
+
+            <!-- desktop version  -->
+        <div class="list_navs">
+            <ul>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#project">Project</a></li>
+                <li><a href="#join">Join</a></li>
+                <li><a href="#sponsor">Sponsor</a></li>
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+            
+        </nav>
+         <!-- head content  -->
+        <div class="head_content">
+            <h2>Find that <br>elusive Unicorns</h2>
+        </div>
+        <div class="header_quote">
+            <p>We at Sano have been in business since 2018, we have assist over millions of people find the right unicorn to invest into . We discover Unicorns with potential of becoming a successful startups and help them look for potential investors and also help investors by giving them the good investment opportunity.</p>
+        </div>
+        <div class="invitation">
+            <p>Please contact us through email for futher </p>
+            <p>questions about SANO</p>
+            <p class="underline"><strong>support@sano.org</strong></p>
+        </div>
+    </div>
+     <!-- about section  -->
+    <div class="logo" id>    
+        <div class="logo_content">
+            <h4>ABOUT SANO LOGO</h4>
+            <img src="./images/line2.png" alt="line" width="160px" height="20px">
+            <p class="logop"><strong>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta molestias nam similique numquam amet magni,<br> 
+                voluptates facere consequuntur<br> 
+                fugit aliquid illo incidunt quod accusantium
+                 laudantium nesciunt. Itaque, quis. Temporibus, 
+                 ad!
+            </strong></p>
+                 <img class="aboutlogo" src="./images/thelogo (1).png" alt="logo" width="250px" height="250px">
+        </div>    
+    </div>
```

Comment:
- [ ] Make sure that the about page looks exactly like the provided design. kindly consider changing the background image to something similar to the CC design, so your page looks exactly as the provided design

|Your Page|CC design|
|-|-|
|![Screenshot_20221123_202523](https://user-images.githubusercontent.com/25227022/203630944-79f20d67-11cc-45f5-aa85-8153c7a7f3e3.png)|![Screenshot_20221123_202842](https://user-images.githubusercontent.com/25227022/203631115-15d00294-36b0-4453-bff4-0f0c34f2e833.png)|